### PR TITLE
Improvement: Marketing button HTML optimization.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- Task - https://app.startinfinity.com/b/Mi5zu3VRVzk/iHKt2gD8SSQ/ea592036-54b9-4155-9907-545aa011ba50?view=0a6f113f-f5c3-4b4e-82f0-1a6fc4831d3e&t=attributes
- Removed unnecessary classes and div from the marketing button.
- Manage remove classes CSS in style.scss.
- Doc for changes - https://docs.google.com/document/d/1ZNsQoG4eEtac9_bxWwwt6XBkOSdfarQ3D9vaulpcss0/edit

### Screenshots

- https://share.getcloudapp.com/xQujJPOd

### Types of changes

- Improvement

### How has this been tested?

Test case 1 - Backward compatibility.
Test case 2 - Check all settings are working or not.
Special Test case :
 - Heading/ Description normal/hovers color and Typography.
 - Marketing button Background color.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]
Still can see too much nesting of HTML elements, - https://share.bsf.io/jkuEpyO7

**Comment by AbhijitNage123** [comment]
@RajkiranBagal Updated HTML as discuss - https://share.getcloudapp.com/xQujJPOd

**Review by RajkiranBagal — APPROVED** [review]

**Comment by akshaylakh** [comment]
Backward Compatibility-  Alignement is disturbed in editor - https://share.bsf.io/kpu6rZLx
Style > Heading > Bottom spacing not working

**Comment by akshaylakh** [comment]
> @AbhijitNage123
> * Backward Compatibility-  Alignment is disturbed in editor - https://share.bsf.io/kpu6rZLx  -- (fixed)
> * Style > Heading > Bottom spacing not working -- (not working -  https://share.bsf.io/bLudLxew)

**Comment by akshaylakh** [comment]
working fine

**Review by sandeshjangam — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -18545,7 +18545,7 @@
     },
     "through": {
       "version": "2.3.8",
-      "resolved": "https://registry.npmjs.org/through/-/through-2.3.8.tgz",
+      "resolved": "http://registry.npmjs.org/through/-/through-2.3.8.tgz",
```
Comment (sandeshjangam) [inline package-lock.json:18548]:
```suggestion
      "resolved": "https://registry.npmjs.org/through/-/through-2.3.8.tgz",
```

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**